src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1293,6 +1299,161 @@ private function registerPropertyAccessConfiguration(array $config, ContainerBui
+            ;
+            $fetchers[$name] = $id;
+        }
+
+        $routers = array();
+
+        foreach ($config['routers']['directs'] as $name => $router) {
+            if (isset($routers[$name])) {
+                throw new \InvalidArgumentException(sprintf('A router named "%s" already exist.', $name));
+            }
+            if (!isset($fetchers[$router['fetcher']])) {
+                throw new \InvalidArgumentException(sprintf('The router named "%s" could not use fetcher "%s" as it does not exit.', $name, $router['fetcher']));
+            }
+            $id = "worker.router.direct.$name";
+            $container
+                ->register($id, Worker\Router\DirectRouter::class)
+                ->addArgument(new Reference($fetchers[$router['fetcher']]))
+                ->addArgument(new Reference($router['consumer']))
+                ->setPublic(false)
+            ;
+
+            $routers[$name] = $id;
+        }
+
+        foreach ($config['routers']['round_robins'] as $name => $router) {
+            $wrappedRouters = array();
+            foreach ($router['groups'] as $wrappedRouter) {
+                if (!isset($routers[$wrappedRouter])) {
+                    throw new \InvalidArgumentException(sprintf('The router named "%s" could not use "%s" as it does not exit.', $name, $wrappedRouter));
+                }
+                $wrappedRouters[] = new Reference($routers[$wrappedRouter]);
+            }
+            $id = "worker.router.round_robin.$name";
+
+            $container
+                ->register($id, Worker\Router\RoundRobinRouter::class)
+                ->addArgument($wrappedRouters)
+                ->addArgument($router['consume_everything'])
+                ->setPublic(false)
+            ;
+            $routers[$name] = $id;
+        }
+
+        $workers = array();
+
+        foreach ($config['workers'] as $name => $worker) {
+            if (isset($worker['router'])) {
+                if (!isset($routers[$worker['router']])) {
+                    throw new \InvalidArgumentException(sprintf('The worker named "%s" could not use router "%s" as it does not exit.', $name, $worker['router']));
+                }
+                $router = new Reference($routers[$worker['router']]);
+            } elseif ($worker['fetcher']) {
+                if (!isset($fetchers[$worker['fetcher']])) {
+                    throw new \InvalidArgumentException(sprintf('The worker named "%s" could not use fetcher "%s" as it does not exit.', $name, $worker['fetcher']));
+                }
+                $router = new Definition(Worker\Router\DirectRouter::class);
+                $router->addArgument(new Reference($fetchers[$worker['fetcher']]));
+                $router->addArgument(new Reference($worker['consumer']));
+                $router->setPublic(false);
+            }
+
+            $id = "worker.worker.$name";
+            $container
+                ->register($id, Worker\Loop\Loop::class)
```

Comment:
I did not use full import on workflow and I think it's better.
It prefer this way.